ignition/resource_ignition_config_test.go

```diff
@@ -68,6 +68,60 @@ func TestIngnitionFileAppend(t *testing.T) {
 	})
 }
 
+func TestIngnitionFileReplaceNoVerification(t *testing.T) {
+	testIgnition(t, `
+		data "ignition_config" "test" {
+			replace {
+				source = "foo"
+			}
+		}
+	`, func(c *types.Config) error {
+		r := c.Ignition.Config.Replace
+		if r == nil {
+			return fmt.Errorf("unable to find replace config")
+		}
+
+		if r.Source.String() != "foo" {
+			return fmt.Errorf("config.replace.source, found %q", r.Source)
+		}
+
+		if r.Verification.Hash != nil {
+			return fmt.Errorf("verification hash should be nil")
+		}
+
+		return nil
+	})
+}
+
+func TestIngnitionFileAppendNoVerification(t *testing.T) {
+	testIgnition(t, `
+		data "ignition_config" "test" {
+			append {
```

Comment:
That one was copied down from the original tests :)
I've changed them all to tabs so they should look nice.